src/relayclient/GsnTest.ts

```diff
@@ -0,0 +1,57 @@
+import Web3 from 'web3'
+
+import CommandsLogic, { DeploymentResult } from '../cli/CommandsLogic'
+import { GsnDevProvider } from './GsnDevProvider'
+import { configureGSN } from './GSNConfigurator'
+import { DevGSNConfig } from './DevRelayClient'
+import { networks } from '../cli/utils'
+
+interface TestEnvironment {
+  deployment: DeploymentResult
+  devProvider: GsnDevProvider
+}
+
+class GsnTestClass {
```

Comment:
not sure what it is, and why its in the relayclient. if its a test fixture - then move to "test"